website/docs/r/subnet.html.markdown

```diff
@@ -60,10 +60,6 @@ The following arguments are supported:
 
 * `address_prefix` - (Required) The address prefix to use for the subnet.
 
-* `enforce_private_link_service_network_policies` - (Optional) Enable or Disable network policies on the `private link service` in the subnet. Default is `false`.
-
--> **NOTE:** Network policies like network security groups (NSG) are not supported for the private link service. In order to deploy a private link service on a given subnet, an explicit disable setting is required on that subnet(e.g. `enforce_private_link_service_network_policies` = `true`). This setting is only applicable for the private link service. For other resources in the subnet, access is controlled based on Network Security Groups (NSG) security rules definition.
```

Comment:
(note for anyone else passing through) this is removed since it's documented below: https://github.com/terraform-providers/terraform-provider-azurerm/pull/5200/files#diff-cebb216a55611aafe1034f892e79a8ffR75-R81